content/src/denylist/Denylist.ts

```diff
@@ -116,3 +129,17 @@ export enum DenylistAction {
   ADDITION = 'addition',
   REMOVAL = 'removal'
 }
+
+export enum OperationStatus {
```

Comment:
Could we give more context on the name? Maybe something like `DenylistSignatureValidationStatus` and `DenylistSignatureValidationResult`?